make/harbor.cfg

```diff
@@ -112,6 +114,11 @@ ldap_group_gid = cn
 #The scope to search for ldap groups. 0-LDAP_SCOPE_BASE, 1-LDAP_SCOPE_ONELEVEL, 2-LDAP_SCOPE_SUBTREE
 ldap_group_scope = 2
 
+# ------------------------------------------------------
+# The membership attribute name AD/openldap=memberof Oracle=ismemberof
```

Comment:
Do you mind tweaking this to a bit more clear? Something like this:

The membership attribute for AD and OpenLDAP is `memberof`, while Oracle LDAP implementations leverages `ismemberof`.

Reply:
oki, perfect.